.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,23 @@
+name: Continuous Integration 
+on: 
+  push:
+    paths-ignore:
+      - 'README.*'
+      - 'documents/**'
```

Minor, but this folder is actually called `documentation`!
```suggestion
      - 'documentation/**'
```